cache/memoize_cache.go

```diff
@@ -56,6 +57,12 @@ func NewMemoizeCache(rc redis.UniversalClient, defaultExpiration, cleanupInterva
 	return mc
 }
 
+// Scan all the keys for given key
+func (m memoizeCache) Scan(cxt context.Context, key string) (map[string]string, error) {
+	//TODO
+	return m.Cache.Scan(cxt, key)
+}
```

Comment:
I don't think we need this here, because `Cache` is embeded the memoizeCache already has a `Scan` method that will call the KeyVal cache Scan method without us having to explicitly do it here. 

https://goplay.space/#rewq-aIf3fP